src/index.ts

```diff
@@ -6,9 +6,16 @@ import to from 'await-to-js';
 import { App } from '@octokit/app';
 import Octokit from '@octokit/rest';
 
-const { isCi, ...env } = envCi();
-const [owner = process.env.OWNER || '', repo = process.env.REPO || ''] =
-  'slug' in env ? env.slug.split('/') : [];
+const { isCi } = envCi();
+
+async function getRepositoryParameters() {
+  const regex = /https?:\/\/.*\/(.+)\/(.+)\.git/;
+  const url = (await execa('git', ['rev-parse', 'HEAD'])).stdout;
+  const [, owner = process.env.OWNER, repo = process.env.REPO] = url.match(regex) || [];
```

Comment:
I'd still like it to fall back to `env` for build servers that might not set a git url by default. I know this sounds crazy but there are some envs where this is the case.